### ISSUE TITLE
The great metastation issues list (at least ones i found)

### ISSUE THREAD

**Issue by Abopos** [wall-nerds/wallening#275]
Small list all issues i found on meta station - Some of them are due to sprites being not finished but some might need to be corrected in map editors - I am running metastation that comes by downloading from this repo - Now no more yapping and to the issues

I'll be writing what i think the issue is in parentheses

**1.bridge axe is inaccessible without alt click (no south sprite) - simply can't get it without alt click**
![image](https://github.com/user-attachments/assets/f841d6f8-745b-429d-919b-23c7a23da17b)

**2.heads offices buttons are misaligned (bad mapping/missing sprites?) - rd,hop's office buttons for shutters etc. are incorrectly stacked/inaccessible**
![image](https://github.com/user-attachments/assets/bf936981-e366-42c0-8921-f4d45691a9bc)
![image](https://github.com/user-attachments/assets/e29e6e54-3c1f-4b62-ba30-1e379375c6ae)

**3.tinted windows are NOT WORKING (bad code?) - for example robo windows shouldn't be see through**
![image](https://github.com/user-attachments/assets/184ab8cb-f3d7-4190-8419-991258d98c87)

**4.CMO office shutter button+keycard panel is literally unclickable without alt click (missing sprites for south?) - and its also invisible so you gotta remember where it was**
![image](https://github.com/user-attachments/assets/8ceb06f7-208e-47a1-818f-f0628df49b2a)

**5.Across the station showers+sinks are incorrectly rotated(i think its an issue already but bad mapping) - basically none of the sinks/showers are correct**
![image](https://github.com/user-attachments/assets/13528186-0932-46dd-92a1-96287b4ba877)
![image](https://github.com/user-attachments/assets/70c53091-03b0-4807-8105-1438169e7f14)

**6.you cant enter/exit xenobiology (bad mapping/sprite issue?) - the button is just nowhere there, you can't alt click it no pressing so doors are just bolted till ai or rd open's them - to be precise we are talking about outer door - also control panel sticks out weirdly inside**
![image](https://github.com/user-attachments/assets/cbfb4f16-64d2-456f-86cf-6eb71b62a0b2)
![image](https://github.com/user-attachments/assets/27399c94-814e-4fa3-a095-78fdb94a648d)

**7.ai upload has big error? (no clue what the fuck is that) - as image shows upload has that idk what that is - also the yellow lights are flashing through the walls**
![image](https://github.com/user-attachments/assets/24e8cfe4-f762-432b-8eb6-9ae330d5655e)

**8.lavaland dorms sign is not sprited i think? just an error**
![image](https://github.com/user-attachments/assets/1f4e3175-0f93-43d2-8019-8829517ca026)

**9.Awful looking aux base (sprites+mapping issue?) - they just look bad no words needed**
![image](https://github.com/user-attachments/assets/307a486b-b6ba-46f8-8769-c44488c5c8aa)

**10.teleporter room shutter control is inaccessible (mapping issue) - you can't press it, its inside the shutters**

**11.The kitchen firelocks cover like half of the table (sprite issue)- yeah like half of the table is served to the firelocks(THEY EATING GOOD TONIGHT)**
![image](https://github.com/user-attachments/assets/6a07b5f2-391b-43c9-8011-290eb75d1197)

so to sum it up - some stuff is missing sprites (mostly south walls) - other is missplaced/missing - and there are probably 5 times more issues than listed here - note that i didn't list obvious issues that you can see in 5 seconds after looking at the station(signs missaligned, most wall items missaligned, VERY fucked up offset etc.)

Now **warning** an opinion of my own about everything i have seen/heard so far

So the sprites, i cant lie they look good. Lots of work was put into individual ones you can see it in everything around you. But certain sprites are ... just straight up making everything look ugly and out of place. And the horrible offset everywhere makes it disorientating to play sometimes(most of the time). Overall if we look from far away wallening is looking good, clean sprites few small errors and its ready, but moment i start to play/walk around it just all feels wrong - some stuff is between huh? to yeah no why is that in the game. IMO the wallening is bad, i don't like it, (at least in its current state) yes it might look pretty if we place few walls on small showcase maps but big stations just break really quickly and just doesn't feel correct and overall... just doesn't **click** together - i know my words have no meaning, but i still ask you to delay the wallening by few more months till you finish it (or cancel but we all know that wont happen).

**Comment by Abopos** [comment]
probably some issues i pointed out already exist - so just link em and ill edit them out

**Comment by TheVekter** [comment]
Tinted windows is a code problem I think, someone else noticed that in another issue thread.

I'll go through these this weekend and fix what I can, most of this is just "move it elsewhere".

**Comment by TheVekter** [comment]
Okay let me go over what I can/can't/have/haven't fixed:

1) I fixed this, it's on the west wall of the bridge now.
2) These are due to button offsets being kind of broken right now, I'm waiting on StrongDMM getting fixed or us figuring out what the hell's going on.
3) Code issue.
4) See 2.
5) Not fixed but I will be fixing it.
6) See 2.
7) Fixed.
8) Probably a missing sprite, out of my wheelhouse.
9) Can and will fix.
10) I haven't looked at this but probably also buttons.
11) This is just because the current implementation of firelocks sucks and people are considering how to fix it.
